### PR TITLE
KS-121 Added a simple check for blank PTOK value.

### PR DESCRIPTION
**Pull request by preslavPA**
Added a simple check for a blank PTOK value in hashPaymentToken method, after a brief discussion with boyan on how this is implemented in the java sdk and noticed that such functionality is missing from the php sdk.  @gallilama this simple fix sorts out [KS-121](https://kopana.atlassian.net/browse/KS-121) as far as I can tell from my testing with a direct RIS call.

### CODE REVIEW

File: src/Kount/Util/Khash.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ public static function getSaltPhrase() {
   public static function hashPaymentToken ($token) {
     $firstSix = mb_substr($token, 0, 6, 'latin1');
     $hash = self::hash($token, 14);
```
Comment (boyankukushev) [inline src/Kount/Util/Khash.php]: are `mb_substr` and `hash` null-safe in terms of received string value? what is going to return `mb_substr(null, 0, 6, 'latin1');`? `null` or an error will be thrown?

Comment (preslav-anev) [inline src/Kount/Util/Khash.php]: Yes, they are. mb_substr allows null values since php 5.4.18 and the hash function in it's current implementation will also not throw an error if it is provided with a null value. I've confirmed this through tests.

Comment (boyankukushev) [inline src/Kount/Util/Khash.php]: Please, add a simple test demonstrating that trying to khash a null value would return an empty string. Thanks!
